rest-tests/src/test/resources/features/check-operator-version.feature

```diff
@@ -0,0 +1,17 @@
+# @sustainer: acadova@redhat.com
+
+@rest
+
```

Comment:
Please, remove empty lines between @rest annotation and the Feature definition. In case the Feature file contains only one scenario, you can also move @check-operator-version annotation here.